service/history/replication/executable_task_test.go

```diff
@@ -478,6 +484,109 @@ func (s *executableTaskSuite) TestResend_Error() {
 	s.False(doContinue)
 }
 
+func (s *executableTaskSuite) TestImport_Success() {
```

Comment:
you should also add a additional cases, which exercise the import function within the iteration

(TestImport_Success only invoke import function outside the iteration?)

Reply:
Sure. Will add more unit tests in next revision.